### PR TITLE
Create product configuration files for Red Hat Virtualization

### PR DESCRIPTION
**Pull request by poncovka**
Create new files for products Red Hat Virtualization and oVirt Node Next.

(based on commits b497482 and 9a06e82)

### CODE REVIEW

File: pyanaconda/core/configuration/storage.py [hunk 1]
```diff
@@ -87,6 +98,31 @@ def default_partitioning(self):
...
+
+    def _convert_partitioning_scheme(self, value):
+        """Convert the given value into a partitioning scheme."""
+        try:
+            member = PartitioningScheme.__members__[value]
+            return member.value
+        except KeyError:
+            pass
+
+        raise ValueError("'{}' is not a valid partitioning scheme".format(value))
```
Comment (jkonecny12) [inline pyanaconda/core/configuration/storage.py:126]: Please move this into the enum as method. I don't think we should access ``__member__`` private member out of the class.

File: data/anaconda.conf [hunk 2]
```diff
@@ -174,6 +174,64 @@ default_scheme = LVM
...
+# Specify a list of mount points.
+must_not_be_on_root =
+
+# Mount points that are recommended to be reformatted.
+#
+# It will be recommended to create a new file system on a mount point
+# that has a whitelisted prefix, but doesn't have a blacklisted one.
+# Specify lists of mount points.
+reformat_whitelist = /boot /var /tmp /usr
+reformat_blacklist = /home /usr/local /opt /var/www
```
Comment (jkonecny12) [inline data/anaconda.conf:232]: What if something is not on whitelist neither blacklist? What then will happen?
Also why `/` is not specified here?

File: pyanaconda/core/configuration/storage_constraints.py [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    def must_be_on_linuxfs(self):
+        """Mount points that must be on a linux file system.
+
+        :return: a set of mount points
+        """
+        return set(self._get_option("must_be_on_linuxfs").split())
+
+    @property
+    def must_be_on_root(self):
+        """Mount points that must be on the / file system.
```
Comment (jkonecny12) [inline pyanaconda/core/configuration/storage_constraints.py:125]: Shouldn't it be:
``Mount points that must be on the / partition.``

IMHO FS is in most cases used for format used on the storage device but this is not what we are talking here.

File: pyanaconda/core/configuration/storage.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/core/configuration/storage.py:126]: Fixed.

File: data/anaconda.conf (hunk 2 above)
Comment (poncovka) [reply data/anaconda.conf:232]: > What if something is not on whitelist neither blacklist? What then will happen?

We will not show a message with the recommendation. Is it really so surprising?

> Also why `/` is not specified here?

We require to reformat the file system on `/`. It is a different check.

File: pyanaconda/core/configuration/storage_constraints.py (hunk 3 above)
Comment (poncovka) [reply pyanaconda/core/configuration/storage_constraints.py:125]: Fixed.

**Comment by poncovka** [comment]
Rebased and fixed.

File: pyanaconda/core/configuration/storage_constraints.py (hunk 3 above)
Comment (VladimirSlavik) [reply pyanaconda/core/configuration/storage_constraints.py]: I know I'm late to the party, but is device possibly a good word here?

Comment (poncovka) [reply pyanaconda/core/configuration/storage_constraints.py]: I have changed it to:
`Paths that must be directories on the / file system.`

File: data/anaconda.conf (hunk 2 above)
Comment (jkonecny12) [reply data/anaconda.conf:232]: OK, I guess this should be enough.

**Comment by poncovka** [comment]
Rebased and tested.

**Pull request merged**